web_src/less/_repository.less

```diff
@@ -33,8 +33,8 @@
             white-space: nowrap;
         }
 
-        .svg.octicon-repo-forked {
-            margin-top: -1px;
+        .svg {
+            margin: 0 .42857143em 0 -.21428571em;
```

Comment:
Why these strange values? Are they based on fomantic's?

Reply:
Yes

Reply:
Additionally this fixes lack of margin on right side between octicon and text by using same margin as icons used.

Reply:
Ok. I'd try to avoid negative margins if possible, thought.

Reply:
I'd prefer to use Fomantic styling here for consistency sake.